ios/Codeforces Watcher/Application/features/auth/signUp/SwitchToSignUpView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  SwitchToSignUpView.swift
+//  Codeforces Watcher
+//
+//  Created by Ivan Karavaiev on 2/1/21.
+//  Copyright © 2021 xorum.io. All rights reserved.
+//
+
+import UIKit
+
+class SwitchToSignUpView: UIView {
```

Comment:
You can make it more universal, so it accepts hint and switch text. You can call it `ActionableLabel`, for example.